src/components/AllArticles.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect,useState } from "react"
+import axios from "axios"
+import { Link } from "react-router-dom"
+
+function AllArticles({setArticleChoice}){
+    const [articlesArr, setArticlesArr] = useState([])
+    const [loading, setLoading] = useState("loading")
+    useEffect(()=>{
+        axios.get("https://northcoders-news-api-bjpy.onrender.com/api/articles")
+        .then((articles)=>{
+            setArticlesArr(articles.data.articles)
+            setLoading("loaded")
+        })
+    },[])
+    
+    function handleClick(e){
+        setArticleChoice(e)
+    }
+    if(loading==="loaded"){
+        return (
+            articlesArr.map((article,index)=>{
+                return (
+                    <Link to={`/articles/${article.article_id}`}  key={article.article_id}className="article">
```

Comment:
good work, wrapping it all in a link allows the user to click anywhere on the article card to get to the individual 👍